SwiftUI_DZ/SwiftUI_DZ/ProductView/ProductView.swift

```diff
@@ -120,14 +144,16 @@ struct ProductView: View {
                 .fontWeight(.heavy)
         }
     }
-
+    
     private var recommendBuyHStackView: some View {
         VStack(spacing: 20) {
             HStack {
                 Spacer().frame(width: Constants.spacerWidthValue)
-                Image(Constants.heartImageName)
+                Image(systemName: shopViewModel.isHeartImageFill ? "heart.fill" : "heart")
```

Comment:
```suggestion
                Image(systemName: shopViewModel.isHeartImageFill ? "heart.fill" : "heart")
```
название картинок в константы